### PR TITLE
Added a section for optional params for pickup type/order origin

### PR DESCRIPTION
**Pull request by jkim628**
@davidhelms

### CODE REVIEW

File: doc/custom-partner-integrations.md [hunk 1]
```diff
@@ -22,4 +22,10 @@ Below is a list of order and customer parameters (some required) that we typical
 | Vehicle Information | Information displayed on the FlyBuy dashboard |
 | Pickup Time/Window | Information displayed on the FlyBuy dashboard |
 | Order Contents | Orders that contain specific items can be tagged with custom icons displayed on the dashboard (e.g. fries, frozen items) |
-| Order Status | Allows you to update FlyBuy to reflect any order status changes on your end (e.g. created, ready, delayed, cancelled, completed) |
\ No newline at end of file
+| Order Status | Allows you to update FlyBuy to reflect any order status changes on your end (e.g. created, ready, delayed, cancelled, completed) |
+
+## Recommended Parameters
```
Comment (jnebeker) [inline doc/custom-partner-integrations.md]: Any reason this is a separate "recommended parameters" section?

Comment (jkim628) [reply doc/custom-partner-integrations.md]: Oops, good catch! Meant to say `Optional Parameters`. My thinking was that these are not really recommended for _most_ partners. Thoughts on this? Either way, needs a fix.

Comment (jnebeker) [reply doc/custom-partner-integrations.md]: That sounds good to me!

Comment (jkim628) [reply doc/custom-partner-integrations.md]: This has been updated.

**Pull request merged**